code/modules/surgery/encased.dm

```diff
@@ -213,6 +219,10 @@
 		return
 	var/datum/organ/external/affected = target.get_organ(target_zone)
 
+	if(!istype(tool, /obj/item/tool/bonegel))
```

Comment:
It's so anti-oop
Make a surgery sound var if you want this

Reply:
what do you recommend doing for things like the hemostat which use two different sounds depending on what the surgery step is doing?

Reply:
There are basically two cases:

1. Situations where the sound depends on the tool being used
2. Situations where the sound depends on the action being done

Fortunately these never overlap. For situations where tool matters, use the toolsound
In situations where it's the action that matters and it always plays the same sound, don't use the toolsound, just use the ogg directly.

So for example, at this line in the code you should use the toolsound, but at L111 you should keep it the same where it calls hemostatdig no matter what tool is used. (You might make the name more generic like surgerydig)

Reply:
sounds good! i will do that.